src/components/Spinner/index.jsx

```diff
@@ -0,0 +1,17 @@
+import { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+@withStyles({
+  center: {
+    textAlign: 'center',
+  },
+})
+export default class Spinner extends PureComponent {
+  render() {
+    const { classes } = this.props;
+    const progress = <CircularProgress color="primary" thickness={7} />;
+
+    return <div className={classes.center}>{progress}</div>;
```

Comment:
No need to put CircularProgress in a constant. Also, the default value of the `color` prop is already set to primary.

```jsx
return (
  <div className={classes.center}>
    <CircularProgress thickness={7} />
  </div>
);
```